test/erlcloud_ddb2_tests.erl

```diff
@@ -1225,6 +1290,231 @@ create_table_input_tests(_) ->
+            \"StreamEnabled\": true,
+            \"StreamViewType\": \"NEW_AND_OLD_IMAGES\"
+         },
+         \"TimeToLiveDescription\": {
+            \"AttributeName\": \"none\",
+           \"TimeToLiveStatus\": \"DISABLED\"
+         }
+      }
+   }
+}",
+ input_tests(Response, Tests).
+
+delete_backup_output_tests(_) ->
+ Tests =
+  [?_ddb_test(
+   {"DeleteBackup example response", "
+    {
+    \"BackupDescription\": {
+      \"BackupDetails\": {
+         \"BackupArn\": \"arn:aws:dynamodb:us-east-1:387047610112:table/Forum/backup/01523517555423-69b67bcb\",
+         \"BackupCreationDateTime\": 1523269031.475,
+         \"BackupName\": \"Forum_Backup\",
+         \"BackupSizeBytes\": 1,
+         \"BackupStatus\": \"DELETED\"
+      },
+      \"SourceTableDetails\": {
+         \"ItemCount\": 0,
+         \"KeySchema\": [
+            {
+               \"AttributeName\": \"id\",
+               \"KeyType\": \"HASH\"
+            }
+         ],
+         \"ProvisionedThroughput\": {
+            \"ReadCapacityUnits\": 5,
+            \"WriteCapacityUnits\": 5
+         },
+         \"TableArn\": \"arn:aws:dynamodb:us-east-1:387047610112:table/Forum\",
+         \"TableCreationDateTime\": 1523269031.475,
+         \"TableId\": \"284a7e42-42ed-4854-b9fd-9a5a4b972dc4\",
+         \"TableName\": \"Forum\",
+         \"TableSizeBytes\": 1
+      },
+      \"SourceTableFeatureDetails\": {
+         \"GlobalSecondaryIndexes\": [
+            {
+               \"IndexName\": \"Idx\",
+               \"KeySchema\": [
+                  {
+                     \"AttributeName\": \"Atr\",
+                     \"KeyType\": \"HASH\"
+                  }
+               ],
+               \"Projection\": {
+                  \"NonKeyAttributes\": [ \"Atr2\" ],
+                  \"ProjectionType\": \"ALL\"
+               },
+               \"ProvisionedThroughput\": {
+                  \"ReadCapacityUnits\": 5,
+                  \"WriteCapacityUnits\": 5
+               }
+            }
+         ],
+         \"LocalSecondaryIndexes\": [
+            {
+               \"IndexName\": \"Idx2\",
+               \"KeySchema\": [
+                  {
+                     \"AttributeName\": \"Atr3\",
+                     \"KeyType\": \"RANGE\"
+                  }
+               ],
+               \"Projection\": {
+                  \"NonKeyAttributes\": [ \"Atr4\" ],
+                  \"ProjectionType\": \"KEYS_ONLY\"
+               }
+            }
+         ],
+         \"SSEDescription\": {
+            \"Status\": \"DISABLED\"
+         },
+         \"StreamDescription\": {
+            \"StreamEnabled\": true,
+            \"StreamViewType\": \"NEW_AND_OLD_IMAGES\"
+         },
+         \"TimeToLiveDescription\": {
+            \"AttributeName\": \"none\",
+           \"TimeToLiveStatus\": \"DISABLED\"
+         }
+      }
+   }
+}",
+    {ok,#ddb2_backup_description{
+     backup_details =
+     #ddb2_backup_details{
```

Comment:
Please indent this in from the record field it's bound to. 4 spaces seems good, like what's used in most of the other tests. At least some indentation this is kinda hard to read at a glance.